client/src/pages/Add.jsx

```diff
@@ -11,17 +11,27 @@ const Add = () => {
   const { id } = useParams();
   const [currentBoulder, setCurrentBoulder] = useState();
   const [climbedBoulder, setClimbedBoulder] = useState();
+  const [formPrefilledClimbedBoulder, setFormPrefilledClimbedBoulder] =
+    useState({});
   const [error, setError] = useState(false);
 
   const fetchCurrentBoulder = () => {
-    fetch(`${URL}/api/add/${id}/`)
+    fetch(`${URL}/api/add/9999/${id}/`)
```

Comment:
Why would you like to have such a hard coded number in your url?

Reply:
I put it there as a reminder for when I set up user login to have different users. For now I use a hard coded user id. But when there a more than one user I'm going to exchange it with the user id. Would that be accepted?